.github/workflows/sdkversion.yml

```diff
@@ -14,6 +14,6 @@ jobs:
         uses: actions/checkout@v3
       - name: Verify sdkVersion is not empty and is not existing tag
         run: |
-          git fetch -a
+          git fetch -a -t
```

Comment:
this was necessary since the github action was giving false-negatives (since not all tags were being fetched)